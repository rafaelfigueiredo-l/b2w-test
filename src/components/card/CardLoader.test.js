import CardLoader from './CardLoader';

describe('<CardLoader />', () => {
    it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<CardLoader />, div);
      ReactDOM.unmountComponentAtNode(div);
    });

    it('Snapshot testing', () => {
      const wrapper = shallow(<CardLoader />);
      expect(wrapper).toMatchSnapshot();
    });
});