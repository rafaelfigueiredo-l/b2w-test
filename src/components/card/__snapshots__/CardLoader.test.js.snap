// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardLoader /> Snapshot testing 1`] = `
<styled.div>
  Loading...
</styled.div>
`;
